core/src/main/spotbugs/exclude-filter.xml

```diff
@@ -15,6 +15,107 @@
   limitations under the License.
 -->
 <FindBugsFilter>
+  <Match>
```

Comment:
Could possibly use @SuppressFBWarnings java annotation instead of these xml exclusions.

Reply:
I really want to do that. I'm going to look into this after this PR is done with, as a separate thing.

Reply:
Yeah that could be a follow on - replace all exclude-filter.xml with annotations.